### DISCUSSION TITLE
error Module Not Found

### DISCUSSION THREAD

**Discussion by flash168** [sebastienros/jint#1476] [Q&A]
` var engine = new Engine(cfg =>cfg.EnableModules(Js_path));
var ns = engine.ImportModule("./js/drpy.min.js");`
has error "System.ArgumentException:“Module Not Found:  Arg_ParamName_Name"
[drpy.min.js ](https://gitcode.net/qq_32394351/dr_py/-/raw/master/libs/drpy.min.js)
Please help me

**Comment by Genteure** [comment]
I took a quick look but didn't find any reference to `Arg_ParamName_Name` in the link you posted, are you sure your local copy is exactly the same as the link you posted?
The project you linked seems to be using `js2py` to translate javascript into python then running in a python environment, unless you are able to replicate the exact same API provided by their custom runtime in C#, I don't think you can run that script in Jint as is. For example, they were calling `request(...)` to send network requests, that is not a standard javascript API.
Please use discussions for general usage questions instead of issues.

-----

简单看了一下脚本内容，没有找到 `Arg_ParamName_Name`，你确定你本地运行的文件内容和你发的链接里的内容完全一样吗？
你发的这个项目看起来是用 `js2py` 把 javascript 翻译成 python 后运行的，除非你能在 C# 里重新实现一套和他的运行时提供的完全一样的 API，我不觉得你能用 Jint 直接运行那个脚本。举例，脚本里调用了 `request(...)` 发送网络请求，这不是标准 javascript API。
一般使用方法的问题麻烦用 discussions 而不是 issues，你发的上一个 issue 就是被转到 discussions 里了的。
